### ISSUE TITLE
Remote URL job options do not update when the refresh button is selected

### ISSUE THREAD

**Issue by ajxb** [rundeck/rundeck#2024]
I believe the following to be a regression in functionality between 2.6.8 and 2.6.9.

For any job that has job options that make use of remote URLs it used to be possible to update the option values by clicking on the tick icon displayed just before the option name on the job page (assuming that the URL sometimes returns different values).

This functionality was working in 2.6.8, however on upgrading to 2.6.9 it no longer works, the only way to refresh the data from a remote URL seems to be to refresh the page in the browser.

**Comment by gschueler** [comment]
what browser/OS are you using?

**Comment by gschueler** [comment]
i think the target that sees the click is only the option name now, not the entire label section.  If I click the option name itself it does reload for me, but not sure if it is a browser issue for you

**Comment by ajxb** [comment]
I'm using Windows 10 and have seen this behaviour in both FireFox 48.0.2 and Chrome 53.0.2785.89 m.

I've tried clicking on the tick icon, and the option name but the values are not refreshed from the source URL.  I have double checked and the data being returned by my service has indeed changed.  When I hover over any part of the label / tick icon the icon change to the refresh icon, but clicks do not seem to register.

I'm not sure that there is anything particularly special about my options, or the data that is being sent.  I've attached an example of a job that exhibits this behaviour and the data driving the scripts.version option.

[data.json.txt](https://github.com/rundeck/rundeck/files/455274/data.json.txt)
[job.yaml.txt](https://github.com/rundeck/rundeck/files/455275/job.yaml.txt)
